tests/test-class_info.php

```diff
@@ -55,15 +55,10 @@ function test_postCountByImportStatus($assert = true)
 		if($assert)
 		{
 			$this->assertEquals(2, $arrPostCount[0]->entryCount);
-			$this->assertEquals("", $arrPostCount[0]->pinged);
-		}
-
-		$import->index_searchstrings();
-		if($assert)
-		{
-			$this->assertEquals("", $arrPostCount[0]->indexed);
```

Comment:
Removed this for a couple reasons. First, `indexed` is not one of the fields returned `postCountByImportStatus()`. Second, `$import->index_searchstrings();` does not modify `$arrPostCount`.